BioArchLinux/rstudio-desktop/PKGBUILD

```diff
@@ -0,0 +1,138 @@
+# Maintainer: Artem Klevtsov <a.a.klevtso@gmail.com>
+# Maintainer: Stephen Martin <hwkiller@gmail.com>
+# Contributor: Conor Anderson <conor@conr.ca>
+
+pkgname=rstudio-desktop
+pkgver=2023.06.2+561
+_pkgver=${pkgver//+/.}
+_srcname=rstudio-${pkgver//+/-}
+_gitcommit=${_get_commit_from_tag}
+_gitname=rstudio-rstudio-${_gitcommit}
+_gwtver=2.8.2
+_ginver=2.1.2
+_nodever=16.14.0
+_pandocver="current"
+_quarto="FALSE"
+pkgrel=1
+pkgdesc="A powerful and productive integrated development environment (IDE) for R programming language"
+arch=('x86_64')
+url="https://posit.co/products/open-source/rstudio/"
+license=('AGPL3')
+depends=('r>=3.3.0' 'boost-libs' 'qt5-sensors' 'qt5-svg' 'glibc' 'zlib' 'bash' 'gcc-libs' 'qt5-webengine' 'qt5-xmlpatterns' 'postgresql-libs' 'sqlite3' 'soci' 'clang' 'hunspell-en_US' 'mathjax2' 'pandoc' 'yaml-cpp' 'r-rmarkdown' 'r-renv' 'r-testthat' 'r-xml2' 'r-yaml')
+makedepends=('git' 'cmake>=3.1.0' 'boost' 'desktop-file-utils' 'jdk8-openjdk' 'apache-ant' 'unzip' 'openssl' 'libcups' 'pam' 'patchelf' 'wget' 'yarn')
+optdepends=('git: for git support'
+            'subversion: for subversion support'
+            'openssh-askpass: for a git ssh access'
+            'quarto: for Quarto projects support')
+
+provides=('rstudio-desktop')
+conflicts=('rstudio-desktop' 'rstudio-desktop-bin' 'rstudio-desktop-preview' 'rstudio-desktop-git')
```

Comment:
The alternative versions in AUR list this package in `conflicts`, so this line is unnecessary as well. See the [wiki](https://wiki.archlinux.org/title/PKGBUILD#conflicts).

Reply:
Done